pytorch_lightning/accelerators/accelerator.py

```diff
@@ -453,7 +456,7 @@ def model_sharded_context(self) -> Generator:
             yield
 
     # todo: remove in v1.5
-    def connect_training_type_plugin(self, plugin: TrainingTypePlugin, model: LightningModule) -> None:
+    def connect_training_type_plugin(self, model: LightningModule) -> None:
```

Comment:
you should keep this signature for BC

Reply:
Isn't this an internal function?

Reply:
@Borda chose to keep it for BC until 1.5 as indicated by the comment right above.

It should be fine to keep the `plugin` argument for `connect_training_type_plugin` and just not pass it to `setup_training_type_plugin`

There's no need to deprecate the `setup_training_type_plugin` signature change as it is unreleased.

Reply:
okay.. Done!